diff --git a/src/Controller/ArticleController.php b/src/Controller/ArticleController.php
index c3f3abd..c8e6147 100644
--- a/src/Controller/ArticleController.php
+++ b/src/Controller/ArticleController.php
@@ -43,8 +43,9 @@ class ArticleController extends AbstractController
     /**
      * @Route("/news/{slug}", name="article_show")
      */
-    public function show($slug, SlackClient $slack, ArticleRepository $articleRepository)
+    public function show($slug, SlackClient $slack, ArticleRepository $articleRepository, $isMac)
     {
+        dump($isMac);
         $article = $articleRepository->findOneBy(['slug' => $slug]);
 
         if (!$article) {
