diff --git a/src/Controller/ArticleController.php b/src/Controller/ArticleController.php
index c173abb..ece958d 100644
--- a/src/Controller/ArticleController.php
+++ b/src/Controller/ArticleController.php
@@ -17,10 +17,14 @@ class ArticleController extends AbstractController
      * Currently unused: just showing a controller with a constructor!
      */
     private $isDebug;
+    private $logger;
 
-    public function __construct(bool $isDebug)
+    public function __construct(bool $isDebug, LoggerInterface $logger)
     {
         $this->isDebug = $isDebug;
+        $this->logger = $logger;
+
+        $this->logger->info('Controller instantiated!');
     }
 
     /**
