diff --git a/src/Controller/ArticleController.php b/src/Controller/ArticleController.php
index 5679f01..ac5a329 100644
--- a/src/Controller/ArticleController.php
+++ b/src/Controller/ArticleController.php
@@ -10,6 +10,7 @@ use Psr\Log\LoggerInterface;
 use Symfony\Component\Routing\Annotation\Route;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\Routing\Exception\NoConfigurationException;
 
 class ArticleController extends AbstractController
 {
@@ -44,7 +45,7 @@ class ArticleController extends AbstractController
         $article = $articleRepository->findOneBy(['slug' => $slug]);
 
         if (!$article) {
-            throw $this->createNotFoundException();
+            throw $this->createNotFoundException('', new NoConfigurationException());
         }
 
         if ($article->getSlug() === 'khaaaaaan') {
