diff --git a/src/Controller/ArticleController.php b/src/Controller/ArticleController.php
index bcde5e3..c3f3abd 100644
--- a/src/Controller/ArticleController.php
+++ b/src/Controller/ArticleController.php
@@ -10,7 +10,6 @@ use Psr\Log\LoggerInterface;
 use Symfony\Component\Routing\Annotation\Route;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\JsonResponse;
-use Symfony\Component\Routing\Exception\NoConfigurationException;
 
 class ArticleController extends AbstractController
 {
@@ -49,7 +48,7 @@ class ArticleController extends AbstractController
         $article = $articleRepository->findOneBy(['slug' => $slug]);
 
         if (!$article) {
-            throw $this->createNotFoundException('', new NoConfigurationException());
+            throw $this->createNotFoundException();
         }
 
         if ($article->getSlug() === 'khaaaaaan') {
