diff --git a/vendor/symfony/http-kernel/EventListener/ErrorListener.php b/vendor/symfony/http-kernel/EventListener/ErrorListener.php
index f5cac76..900a83c 100644
--- a/vendor/symfony/http-kernel/EventListener/ErrorListener.php
+++ b/vendor/symfony/http-kernel/EventListener/ErrorListener.php
@@ -53,6 +53,7 @@ class ErrorListener implements EventSubscriberInterface
             return;
         }
 
+        dd($this->controller);
         $exception = $event->getThrowable();
         $request = $this->duplicateRequest($exception, $event->getRequest());
 
