diff --git a/vendor/symfony/http-kernel/EventListener/ErrorListener.php b/vendor/symfony/http-kernel/EventListener/ErrorListener.php
index 900a83c..f5cac76 100644
--- a/vendor/symfony/http-kernel/EventListener/ErrorListener.php
+++ b/vendor/symfony/http-kernel/EventListener/ErrorListener.php
@@ -53,7 +53,6 @@ class ErrorListener implements EventSubscriberInterface
             return;
         }
 
-        dd($this->controller);
         $exception = $event->getThrowable();
         $request = $this->duplicateRequest($exception, $event->getRequest());
 
