diff --git a/vendor/symfony/http-kernel/Controller/ErrorController.php b/vendor/symfony/http-kernel/Controller/ErrorController.php
index 2a7b724..b6c4401 100644
--- a/vendor/symfony/http-kernel/Controller/ErrorController.php
+++ b/vendor/symfony/http-kernel/Controller/ErrorController.php
@@ -38,7 +38,6 @@ class ErrorController
 
     public function __invoke(\Throwable $exception): Response
     {
-        dd($this->errorRenderer);
         $exception = $this->errorRenderer->render($exception);
 
         return new Response($exception->getAsString(), $exception->getStatusCode(), $exception->getHeaders());
