diff --git a/src/EventListener/UserAgentSubscriber.php b/src/EventListener/UserAgentSubscriber.php
index b099c6d..1a059ed 100644
--- a/src/EventListener/UserAgentSubscriber.php
+++ b/src/EventListener/UserAgentSubscriber.php
@@ -20,6 +20,10 @@ class UserAgentSubscriber implements EventSubscriberInterface
     {
         $request = $event->getRequest();
 
+        $request->attributes->set('_controller', function() {
+            return new Response('I just took over the controller!');
+        });
+
         $userAgent = $request->headers->get('User-Agent');
         $this->logger->info(sprintf('The User-Agent is "%s"', $userAgent));
     }
