diff --git a/src/EventListener/UserAgentSubscriber.php b/src/EventListener/UserAgentSubscriber.php
index 1209159..1a059ed 100644
--- a/src/EventListener/UserAgentSubscriber.php
+++ b/src/EventListener/UserAgentSubscriber.php
@@ -4,6 +4,7 @@ namespace App\EventListener;
 
 use Psr\Log\LoggerInterface;
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\HttpKernel\Event\RequestEvent;
 
 class UserAgentSubscriber implements EventSubscriberInterface
@@ -19,6 +20,10 @@ class UserAgentSubscriber implements EventSubscriberInterface
     {
         $request = $event->getRequest();
 
+        $request->attributes->set('_controller', function() {
+            return new Response('I just took over the controller!');
+        });
+
         $userAgent = $request->headers->get('User-Agent');
         $this->logger->info(sprintf('The User-Agent is "%s"', $userAgent));
     }
