diff --git a/vendor/symfony/http-kernel/EventListener/RouterListener.php b/vendor/symfony/http-kernel/EventListener/RouterListener.php
index 31bbc3b..d1ffb36 100644
--- a/vendor/symfony/http-kernel/EventListener/RouterListener.php
+++ b/vendor/symfony/http-kernel/EventListener/RouterListener.php
@@ -113,6 +113,7 @@ class RouterListener implements EventSubscriberInterface
             } else {
                 $parameters = $this->matcher->match($request->getPathInfo());
             }
+            dd($parameters);
 
             if (null !== $this->logger) {
                 $this->logger->info('Matched route "{route}".', [
