diff --git a/src/Controller/ArticleController.php b/src/Controller/ArticleController.php
index 72de930..de583db 100644
--- a/src/Controller/ArticleController.php
+++ b/src/Controller/ArticleController.php
@@ -8,6 +8,7 @@ use App\Service\SlackClient;
 use Doctrine\ORM\EntityManagerInterface;
 use Psr\Log\LoggerInterface;
 use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
 use Symfony\Component\Routing\Annotation\Route;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\JsonResponse;
@@ -27,13 +28,20 @@ class ArticleController extends AbstractController
     /**
      * @Route("/", name="app_homepage")
      */
-    public function homepage(ArticleRepository $repository, LoggerInterface $logger, $isMac)
+    public function homepage(ArticleRepository $repository, LoggerInterface $logger, $isMac, HttpKernelInterface $httpKernel)
     {
         $logger->info('Inside the controller!');
         $articles = $repository->findAllPublishedOrderedByNewest();
 
+        // manual sub-request example
         $request = new Request();
         $request->attributes->set('_controller', 'App\\Controller\\PartialController::trendingQuotes');
+        $request->server->set('REMOTE_ADDR', '127.0.0.1');
+
+        $response = $httpKernel->handle(
+            $request,
+            HttpKernelInterface::SUB_REQUEST
+        );
 
         return $this->render('article/homepage.html.twig', [
             'articles' => $articles,
