diff --git a/src/EventListener/UserAgentSubscriber.php b/src/EventListener/UserAgentSubscriber.php
index 0b35ba6..cde9b29 100644
--- a/src/EventListener/UserAgentSubscriber.php
+++ b/src/EventListener/UserAgentSubscriber.php
@@ -47,13 +47,11 @@ class UserAgentSubscriber implements EventSubscriberInterface
     private function isMac(Request $request)
     {
         if ($request->query->has('mac')) {
-            yield $request->query->getBoolean('mac');
-
-            return;
+            return $request->query->getBoolean('mac');
         }
 
         $userAgent = $request->headers->get('User-Agent');
 
-        yield stripos($userAgent, 'Mac') !== false;
+        return stripos($userAgent, 'Mac') !== false;
     }
 }
