diff --git a/src/Controller/PartialController.php b/src/Controller/PartialController.php
index 831d389..31c6a12 100644
--- a/src/Controller/PartialController.php
+++ b/src/Controller/PartialController.php
@@ -6,12 +6,13 @@ use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 
 class PartialController extends AbstractController
 {
-    public function trendingQuotes()
+    public function trendingQuotes($isMac)
     {
         $quotes = $this->getTrendingQuotes();
 
         return $this->render('partial/trendingQuotes.html.twig', [
-            'quotes' => $quotes
+            'quotes' => $quotes,
+            'isMac' => $isMac
         ]);
     }
 
diff --git a/templates/partial/trendingQuotes.html.twig b/templates/partial/trendingQuotes.html.twig
index a979104..f9a2fa5 100644
--- a/templates/partial/trendingQuotes.html.twig
+++ b/templates/partial/trendingQuotes.html.twig
@@ -7,4 +7,9 @@
             </p>
         {% endfor %}
     </div>
+
+    {% if isMac %}
+        <hr>
+        <small>BTW, you're using a Mac!</small>
+    {% endif %}
 </div>
