diff --git a/src/Controller/ArticleController.php b/src/Controller/ArticleController.php
index 97fe880..216d0a3 100644
--- a/src/Controller/ArticleController.php
+++ b/src/Controller/ArticleController.php
@@ -37,6 +37,7 @@ class ArticleController extends AbstractController
         $logger->info('Inside the controller!');
         $articles = $repository->findAllPublishedOrderedByNewest();
 
+        /*
         // manual sub-request example
         $request = new Request();
         $request->attributes->set('_controller', 'App\\Controller\\PartialController::trendingQuotes');
@@ -47,6 +48,7 @@ class ArticleController extends AbstractController
             HttpKernelInterface::SUB_REQUEST
         );
         dump($response);
+        */
 
         return $this->render('article/homepage.html.twig', [
             'articles' => $articles,
