diff --git a/src/Controller/ArticleController.php b/src/Controller/ArticleController.php
index 9699d84..7a5b59a 100644
--- a/src/Controller/ArticleController.php
+++ b/src/Controller/ArticleController.php
@@ -37,6 +37,7 @@ class ArticleController extends AbstractController
         $logger->info('Inside the controller!');
         $articles = $repository->findAllPublishedOrderedByNewest();
 
+        /*
         // manual sub-request example
         $request = new Request();
         $request->attributes->set('_controller', 'App\\Controller\\PartialController::trendingQuotes');
@@ -46,6 +47,7 @@ class ArticleController extends AbstractController
             $request,
             HttpKernelInterface::SUB_REQUEST
         );
+        */
 
         return $this->render('article/homepage.html.twig', [
             'articles' => $articles,
