diff --git a/src/Controller/ArticleController.php b/src/Controller/ArticleController.php
index 511aff0..c3f3abd 100644
--- a/src/Controller/ArticleController.php
+++ b/src/Controller/ArticleController.php
@@ -45,7 +45,6 @@ class ArticleController extends AbstractController
      */
     public function show($slug, SlackClient $slack, ArticleRepository $articleRepository)
     {
-        return;
         $article = $articleRepository->findOneBy(['slug' => $slug]);
 
         if (!$article) {
