diff --git a/src/Controller/ArticleController.php b/src/Controller/ArticleController.php
index c3f3abd..511aff0 100644
--- a/src/Controller/ArticleController.php
+++ b/src/Controller/ArticleController.php
@@ -45,6 +45,7 @@ class ArticleController extends AbstractController
      */
     public function show($slug, SlackClient $slack, ArticleRepository $articleRepository)
     {
+        return;
         $article = $articleRepository->findOneBy(['slug' => $slug]);
 
         if (!$article) {
